frontend/pages/team.js

```diff
@@ -28,25 +48,20 @@ const Team = () => {
         <div className='row'>
           <div className='col'>
             <div className='container'>
-              <div className='row'>
-                <ul className={style.ul}>
-                  <div className='text-center'>
-                    <li className={style.li}>
-                      <input type='text' id='person-search-input' placeholder='Search...' />
-                    </li>
-                    <li className={style.li}>
-                      <i className='search-icon'>O</i>
-                    </li>
-                    <li className={`${style.li} ${style.filtermenu}`}>
-                      <select className='filter-select'>
-                        <option value='Default'>Filter</option>
-                        <option value='Class'>Class</option>
-                        <option value='Degree'>Degree</option>
-                        <option value='Group'>Group</option>
-                      </select>
-                    </li>
-                  </div>
-                </ul>
+              <div className='row justify-content-between'>
+                <div className='col col-sm-4'>
+                  <label htmlFor='team-search' className='form-label'>Search</label>
+                  <input type='search' className='form-control' id='team-search' onChange={searchHandler} placeholder='Search by name, class, degree, group' />
+                </div>
+                <div className='col col-sm-4 col-md-3 col-lg-2'>
+                  <label htmlFor='team-filter-select' className='form-label'>Filter by</label>
+                  <select className='form-select' id='team-filter-select' onChange={searchHandler}>
+                    <option value='Default'>None</option>
+                    <option value='Class'>Class</option>
+                    <option value='Degree'>Degree</option>
+                    <option value='Group'>Group</option>
+                  </select>
+                </div>
```

Comment:
I say get rid of (or comment out) this entire section.  It's not currently being implemented, so we should hide this until it is.